logic/scripts/update_token_insight.py

```diff
@@ -321,9 +333,46 @@ def fetch_ogn_transactions():
             db.session.add(tx)
             db.session.commit()
 
+# Fetches wallet balance from API and stores that to DB
+def fetch_wallet_balance(wallet):
+    print "Checking the balance of wallet %s" % (
+        wallet,
+    )
+
+    url = "http://api.ethplorer.io/getAddressInfo/%s" % (wallet)
+    results = call_ethplorer(url)
+
+    contact = db_common.get_or_create(
+        db.session, db_models.EthContact, address=wallet
+    )
+
+    if "error" in results:
+        print("Error while fetching balance")
+        print(results["error"]["message"])
+        raise ValueError(results["error"]["message"])
+
+    contact.eth_balance = results["ETH"]["balance"]
+    contact.transaction_count = results["countTxs"]
+
+    print "ETH balance of %s is %s" % (wallet, results["ETH"]["balance"])
+    if "tokens" in results:
+        contact.tokens = results["tokens"]
+        # update the OGN & DAI balance
+        for token in results["tokens"]:
+            if token["tokenInfo"]["address"] == ogn_contract:
```

Comment:
Does that API always return addresses in lower case ? Would it be worth doing the addresses comparison all in lower case in case in the future we move to a different api?

Reply:
Will do that